sdk/storage/Azure.Storage.Blobs.Batch/src/BatchConstants.cs

```diff
@@ -25,6 +25,8 @@ internal static class BatchConstants
         public const string BatchSeparator = "--";
         public const string HttpVersion = "HTTP/1.1";
 
+#pragma warning disable CA1802 // Use literals where appropriate
```

Comment:
Given the compiler in this case actually should make a const, does it not work to declare this as a const field?

Looks like it works: https://sharplab.io/#v2:EYLgtghgzgLgpgJwD4AECMA2ABCgTFgYSwG8BYAKCyqwGMB7AO1hzQAYsBlLAXiwBIARMQYQwcOgDMAFAQCUAXwB0w0eOkcFigGJ06AgNwV5QA==

Reply:
This feature requires a "preview" lang version.

Reply:
That's unfortunate. Guess that only works for `const` fields, then, since it ends up creating a literal for our `DiagnosticScope` expressions. I remember you told me of this when I questioned the performance of formatting at runtime, which it turned out doesn't happen in that case.